### PR TITLE
openstack-ardana: validate job parameter values

### PR DESCRIPTION
**Pull request by stefannica**
Use `validating-string` JJB parameters and add validation to
a few job parameters that, if not validated properly, could
cause serious problems to the CI.

### CODE REVIEW

File: jenkins/ci.suse.de/openstack-ardana-pcloud.yaml [hunk 1]
```diff
@@ -74,74 +77,89 @@
...
-            The number of swift proxy/account/container/object service nodes in the generated input model.
+            The number of swift proxy/account/container/object service nodes in the generated input model (0-3).
 
             Input model generator scenarios using this parameter: mid-scale-kvm.
 
-      - string:
+      - validating-string:
           name: sles_computes
           default: '2'
+          regex: '[0-5]'
```
Comment (flaviodsr) [inline jenkins/ci.suse.de/openstack-ardana-pcloud.yaml]: For the scale environment we have 200+ nodes

File: jenkins/ci.suse.de/openstack-ardana-vcloud.yaml [hunk 2]
```diff
@@ -93,88 +96,104 @@
...
             Input model generator scenarios using this parameter: mid-scale-kvm.
 
             NOTE: this parameter is used to generate input models. See the 'scenario_name' parameter about
             using one of the available input model generator scenarios.
 
-      - string:
+      - validating-string:
           name: sles_computes
           default: '2'
+          regex: '[0-5]'
```
Comment (flaviodsr) [inline jenkins/ci.suse.de/openstack-ardana-vcloud.yaml]: Same here

File: jenkins/ci.suse.de/templates/cloud-ardana-pipeline-template.yaml [hunk 3]
```diff
@@ -167,88 +174,104 @@
...
             Input model generator scenarios using this parameter: mid-scale-kvm.
 
             NOTE: this parameter is used to generate input models. See the 'scenario_name' parameter about
             using one of the available input model generator scenarios.
 
-      - string:
+      - validating-string:
           name: sles_computes
           default: '{sles_computes|2}'
+          regex: '[0-5]'
```
Comment (flaviodsr) [inline jenkins/ci.suse.de/templates/cloud-ardana-pipeline-template.yaml]: Same here

File: jenkins/ci.suse.de/openstack-ardana-pcloud.yaml (hunk 1 above)
Comment (stefannica) [reply jenkins/ci.suse.de/openstack-ardana-pcloud.yaml]: Right. Good point.

File: jenkins/ci.suse.de/templates/cloud-ardana-pipeline-template.yaml (hunk 3 above)
Comment (stefannica) [reply jenkins/ci.suse.de/templates/cloud-ardana-pipeline-template.yaml]: Done

**Pull request merged**